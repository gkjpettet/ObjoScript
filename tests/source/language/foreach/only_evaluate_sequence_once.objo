function f() {
  System.print("evaluate sequence")
  return [1, 2, 3]
}

foreach i in f() {
 System.print(i)
}
# expect: evaluate sequence
# expect: 1
# expect: 2
# expect: 3