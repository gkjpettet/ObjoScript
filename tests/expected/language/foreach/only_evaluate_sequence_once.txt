evaluate sequence
1
2
3